login-module/test/browser/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
This page can be opened directly in the browser:

![result-mocha](https://user-images.githubusercontent.com/333276/81476514-9714a280-9212-11ea-9561-f9837765bf24.png)